package.nls.ja.json

```diff
@@ -34,16 +33,13 @@
     "config.print.onFileSave.description": "ファイル保存時に現在のドキュメントをHTMLへ出力する",
     "config.print.validateUrls.description": "出力時のURL検証の有効化/無効化",
     "config.print.theme": "出力されたHTMLのテーマ",
-    "config.syntax.decorations.description": "Add syntax decorations",
-    "config.syntax.plainTheme.description": "`extension.syntax.decorations`が有効な場合にのみ作用させる",
     "config.katex.macros.description": "ユーザ定義のKaTeXマクロ",
     "config.completion.root": "Pathの自動補完におけるルートフォルダ",
-    "noValidMarkdownFile": "有効なMarkdownファイルがありません",
-    "printing": "出力中",
-    "to": "へ",
-    "unableToReadFile": "ファイルを読み取れません",
-    "revertingToImagePaths": "base64エンコードを画像Pathに戻す",
-    "customStyle": "カスタムスタイル",
-    "notFound": "が見つかりません。",
+    "ui.exporting.messageCustomCssNotFound": "が見つかりません。 '{0}'",
```

Comment:
Japanese grammar is complex. Without help from a native speaker, I don't dare to write new sentences, but can only copy existing ones.

Reply:
It was a full sentence (split into two parts) 😅

https://github.com/yzhang-gh/vscode-markdown/blob/115935fdfd0acc1c74f6d497a8415f2c8086b30a/src/print.ts#L222

https://github.com/yzhang-gh/vscode-markdown/blob/115935fdfd0acc1c74f6d497a8415f2c8086b30a/package.nls.ja.json#L48-L49

I know a thing or two about the grammar and it looks correct.

Reply:
> It was a full sentence (split into two parts)

Well, I feel that history harms the reliability of #608.

---

According to
https://github.com/microsoft/vscode-loc/blob/e2ae4cfade0152e6d10d27cb80cee78ef1bd2af9/i18n/vscode-language-pack-ja/translations/extensions/markdown-language-features.i18n.json

Perhaps we don't need to emphasis "Custom CSS" or "Custom style sheet".

```
'{0}' が見つかりません。
```

is enough.

Reply:
I agree. I wrote it because I have seen

```
"ui.exporting.messageExportingInProgress": "{1}出力中。 '{0}'",
"ui.general.messageUnableToReadFile": "ファイルを読み取れません。 '{0}'",
```